### PR TITLE
Document the new OCS TOTP Validation API

### LINKED ISSUE #674: Document twofactor_totp key validation OCS API
## What Needs to be Documented?
https://github.com/owncloud/twofactor_totp/pull/38

## Where Does This Need To Be Documented?
- I guess, under Available API's section in developer manual

## What Type Of Content Change Is This? (Optional)
- [x] New Content Addition
- [ ] Old Content Deprecation
- [ ] Existing Content Simplification
- [ ] Bug Fix to Existing Content

## Which Manual Does This Relate To? (Optional)
- [x] Developer Manual

### PR DESCRIPTION
**Pull request by settermjd**
This PR documents the new OCS TOTP Validation API in [2-Factor Authentication](https://marketplace.owncloud.com/apps/twofactor_totp) v0.6.0. The changes show:

- What the API is
- How to use it with a CURL example
- Prerequisites
- Example responses

It fixes #674.

### CODE REVIEW

File: modules/developer_manual/pages/core/apis/ocs-totp-validation.adoc [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+--
+====
+
+=== Returns
+
+The request returns either an XML (the default) or a JSON response, along with an `HTTP 200 OK` status code, which show whether:
+
+. The TOTP is valid 
+. The TOTP is invalid 
+. The user was not found
```
Comment (karakayasemi) [inline modules/developer_manual/pages/core/apis/ocs-totp-validation.adoc]: `The user or secret belongs to the user was not found` more accurate.

File: modules/developer_manual/pages/core/apis/ocs-totp-validation.adoc [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+The status of the TOTP is located in the `ocs/data/result` element.
+If the user was not found, then:
```
Comment (karakayasemi) [inline modules/developer_manual/pages/core/apis/ocs-totp-validation.adoc]: same wıth above.

File: modules/developer_manual/pages/core/apis/ocs-totp-validation.adoc (hunk 1 above)
Comment (settermjd) [reply modules/developer_manual/pages/core/apis/ocs-totp-validation.adoc]: Thanks for pointing that out.

**Comment by settermjd** [comment]
@karakayasemi, thanks for the rapid feedback.

**Pull request merged**